src/tag/index.js

```diff
@@ -22,12 +23,12 @@ class Tag extends PureComponent {
   }
 
   render() {
-    const { label, readOnly } = this.props
+    const { label, readOnly, disabled } = this.props
 
     return (
       <span className={cx('tag')}>
         {label}
-        <button onClick={!readOnly && this.handleClick} className={cx('tag-remove', { readOnly })} type="button">
+        <button onClick={!readOnly && !disabled ? this.handleClick : undefined} className={cx('tag-remove', { readOnly }, { disabled })} type="button">
```

Comment:
Why the explicit `undefined`? I mean React already takes care of it if you do `(!readOnly && !disabled) && this.handleClick` or `!(readOnly || disabled) && this.handleClick`

Reply:
The other expression triggered a warning in the tests. "If you used to conditionally omit it with onClick...". I can change it back.

Reply:
Perhaps related to https://github.com/alexkuz/react-json-tree/pull/97
Not sure if that's in the dependency graph of ours right now.

Reply:
Also se https://stackoverflow.com/a/54038930

Reply:
Well, that's annoying.